draft-irtf-cfrg-frost.md

```diff
@@ -501,7 +504,7 @@ This section describes the subroutine for creating the per-message challenge.
   - group_commitment, an Element in G representing the group commitment
   - group_public_key, public key corresponding to the group signing key, an
     Element in G.
-  - msg, the message to be signed (sent by the Coordinator).
+  - msg, the message to be signed.
```

Comment:
Hm- what is the role of the coordinator then, if we don't say which elements are sent by them?

Reply:
We just don't need to say it here. We do say that the coordinator sends these messages elsewhere.